cmd/tink-worker/action.go

```diff
@@ -61,7 +59,7 @@ func executeAction(ctx context.Context, action *pb.WorkflowAction, wfID string)
 	defer cancel()
 	err = runContainer(timeCtx, l, id)
 	if err != nil {
-		return errRunContainer, 1, errors.Wrap(err, "DOCKER RUN")
+		return 1, errors.Wrap(err, "DOCKER RUN")
```

Comment:
what action state is this? Why isn't this using the symbol?

Reply:
Fixed this one and other places to use symbols and not the magic numbers.